src/flamenco/runtime/context/fd_exec_instr_ctx.h

```diff
@@ -25,7 +25,7 @@ struct __attribute__((aligned(8UL))) fd_exec_instr_ctx {
 
   fd_funk_txn_t * funk_txn;
   fd_acc_mgr_t *  acc_mgr;
-  fd_valloc_t     valloc;
+  fd_valloc_t     valloc; /* Scratch-backed valloc TODO: migrate to transaction spad */
```

Comment:
Can we just set the instr ctx valloc to the spad valloc

Reply:
Some instruction level test harnesses don't give us spads and instead give us libc malloc valloc and etc.  We can do that change in a future PR?  I'll update the comment to reflect that.

Reply:
Okay I renamed this to test_only_valloc and left a comment to make it clear.